scripts/utils/constructProposalCalldata.ts

```diff
@@ -17,17 +20,25 @@ type ExtendedAlphaProposal = {
  * See `proposals/utils/getProposalCalldata.js` on how to construct the proposal calldata
  */
 export async function constructProposalCalldata(proposalName: string): Promise<string> {
-  const proposalInfo = await import(`@proposals/description/${proposalName}`);
+  const proposalInfo = (await import(`@proposals/description/${proposalName}`)).default as ProposalDescription;
 
   const contracts = await getAllContracts();
   const contractAddresses = await getAllContractAddresses();
 
-  const proposal = (await constructProposal(
-    proposalInfo.default,
-    contracts,
-    contractAddresses
-  )) as ExtendedAlphaProposal;
+  const proposal = (await constructProposal(proposalInfo, contracts, contractAddresses)) as ExtendedAlphaProposal;
 
+  console.log(proposals[proposalName].category);
```

Comment:
This console log fails if the proposal isn't in the proposal_config. I don't like the dependency on proposal_config.